app/src/androidTest/java/com/android/sample/ui/profile/EditProfileTest.kt

```diff
@@ -67,13 +69,4 @@ class EditProfileScreenTest {
     composeTestRule.onNodeWithTag("addInterestButton").performClick()
     composeTestRule.onNodeWithTag("interestsList").assertIsDisplayed()
   }
-  /*
```

Comment:
this test could be kept to check that the button indeed navigates to profile screen